---
permalink: /
title: "üëãüèº Hey there, I'm Sohith!"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

üßëüèΩ‚Äçüíª I'm a pre-final year undergraduate student specializing in bioinformatics at **SIMATS, Chennai**.

üßëüèΩ‚Äçüî¨ My research interests lie at the intersection of AI and healthcare, with a focus on neurodegenerative disorders such as Epilepsy, Parkison's, and Alzheimer's, as well as infectious diseases like tuberculosis. Additionally, I am actively engaged in mass spectrometry analysis and systems biology to drive biomedical innovations.

üìö I'm currently working on building an AI-driven platform for early detection of Mycobacterium tuberculosis!

üìà I'm also interested in assisting others on their path in bioinformatics and academia.

Research Experience
====
**Current Experience**

![Research](/sohith/images/ndhu_1.JPG){: .align-right width="150px"}
![Research](/sohith/images/ndhu_2.jpeg){: .align-right width="150px"}
**Research Intern, National Dong Hwa University, Hualien, Taiwan**                                                                                 
*JAN' 25 - JUN' 25*

Operating MALDI-TOF mass spectrometer at [Biophysics Mass Spectrometry Lab, NDHU](http://faculty.ndhu.edu.tw/~PENGW/contact/) for TB and NTM sample analysis under the supervision of [Dr. Wing Peng-Ping](https://www.researchgate.net/profile/Wen-Ping-Peng/research) and developing ML models for tuberculosis classification using acquired spectral data. Performing statistical analysis and feature selection to identify significant m/z peaks, coupled with spectral data visualization for enhanced interpretability.


![Research](/sohith/images/lab_3.png){: .align-right width="150px"}
![Research](/sohith/images/lab_2.png){: .align-right width="150px"}
**Remote Research Assistant, Manipal University College, Melaka, Malaysia**                                                                       
*APR' 24 - PRESENT*

Analyzed Solanum nigrum extract for antibacterial compounds using GC-MS data, successfully identifying 50 bioactive components under the guidance of [Dr. Sugapriya](https://scholar.google.co.in/citations?hl=en&user=dieHes4AAAAJ). This research has laid the groundwork for novel antimicrobial investigations using plant-based compounds.Conducted in silico docking studies on M. tuberculosis MurE ligase using AutoDock Vina to identify potential anti-TB agents from S. nigrum extract. Currently, I am assisting with writing research papers, contributing to book chapters, and supporting various ongoing projects in the field of antimicrobial research.


**Previous Experience**

![Research](/sohith/images/docked.png){: .align-right width="300px"}
**Remote Research Intern, Genomac Hub, Ogbomosho, Nigeria**                                                                                      
*SEP' 24 - NOV' 24*

I worked on selecting peptides with potential anti-cancer activity by analyzing key biochemical properties such as hydrophobicity, net charge, stability, half-life, and Boman index. These properties help determine a peptide's ability to interact with cancer cell membranes, its stability in biological environments, and its targeting specificity. Using correlation heat maps and PCA analysis, our team identified 28 bacterial peptides, 18 probiotic bacterial peptides, 6 fungal peptides, and 4 metagenomic peptides. My specific focus was on peptides from the soil bacterium Streptomyces parvus, contributing to the overall findings of potential anti-cancer candidates. [Graduation Ceremony](https://www.youtube.com/live/Bzw7sYRSYVg?si=28OAN3kkAOeAFobr)


**Undergraduate Research Student, Saveetha School of Engineering, Chennai, India**                                                              
*JAN' 23 - MAR' 24*

Under the supervision of [Dr. Kannan](https://scholar.google.co.in/citations?hl=en&user=RwsuY0gAAAAJ), I conducted data preprocessing and exploratory data analysis (EDA) on the ChEMBL dataset for Acute Myeloid Leukemia‚Äôs drug discovery using pandas and matplotlib/seaborn. I implemented and compared various machine learning models (Random Forest, SVM, XGBoost), achieving an accuracy of 82% with Random Forest for predicting potential drug candidates. Additionally, I performed molecular docking analysis using AutoDock Vina, applied Lipinski's Rule of Five with RDKit, and executed molecular dynamics simulations using GROMACS to validate and analyze protein-ligand interactions of promising Alzheimer's drug candidates.




