---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
B.E. in Bioinformatics, SIMATS University, 2026

* Relevant Coursework: Applied Statistics & Probability, Python, R, Systems Biology, Cell & Molecular Biology,  Recombinant DNA Technology


Research Experience
======
**Remote Undergraduate Student Researcher [University of Colorado Colorado Springs, USA]**		         
*Dec ‘24 - Present*

* Working in collaboration with the University of Colorado and Ioncure to develop and apply AI-driven methods for evaluating neurodegeneration awareness. 
* Leveraging Natural Language Processing (NLP), Computer Vision, Machine Learning, and advanced data science techniques to analyze diverse data sources


**Hybrid Research Assistant [Manipal University College, Malaysia]**                                                 
*Apr ’24 - Present*

* Analyzed S. nigrum extract for antibacterial compounds using GC-MS, identifying 50 bioactive components under Sugapriya's guidance
* Established a foundation for novel antimicrobial research using plant-based compounds.
* Conducted in silico docking studies on M. tuberculosis MurE ligase (2XJA) using AutoDock Vina to identify potential anti-TB agents in S. nigrum extract; discovered anthracene, 9-ethyl-9,10-dihydro as a promising anti-TB compound.
* Applied a systems biology approach to tuberculosis research, identifying key hub genes responsible for the disease. Utilized Cytoscape for network analysis and R programming (with caret, dplyr, ggplot2 packages) for data analysis, machine learning model building, and visualization to complement the wet lab research.
* Collaborated on comprehensive drug discovery and therapeutics reviews with Dr. Sugapriya, integrating her leukemia-focused wet lab research with my dry lab expertise, advancing interdisciplinary research on emerging medical challenges.


**On-site Research Intern [KEK (High Energy Accelerator Research Organization), Japan]**
*Jul '25 - Aug '25*

* Worked at Structural Biology Research Center (SBRC), actively engaged in structural biology research focusing on X-ray crystallography refinement and Cryo-Electron Microscopy (Cryo-EM) analysis of protein structures, under Miki Senda and Toshio Moriya.
* Assisted in the creation of beginner-friendly English manuals and technical documentation to support future international interns and early-stage researchers at KEK, covering common workflows and troubleshooting in macromolecular crystallography.


**On-site Research Intern [National Dong Hwa University, Taiwan]**
*Jan '25 - Jun '25*

* Worked at Biophysical Mass Spectrometry (BMS) lab, designed a high-performance ML pipeline (AUROC: 0.98, F1: 0.94) for TB/NTM diagnosis using MALDI-TOF mass spectrometry data and advanced peak feature engineering.
* Handled MALDI-TOF instrumentation and applied nanodiamond-based enrichment for sputum sample analysis. And, Performed statistical analysis and m/z feature selection, improving diagnostic interpretability and external validation accuracy (90%).


**Remote Research Intern [Joint Institute for Nuclear Research, Russia]**
*Mar '25 - Apr '25*

* Performed atlas-guided virtual histological analysis of irradiated CNS tissue using ImageJ and the Allen Brain Atlas, classifying neurons, glia, and apoptotic cells.
* Automated quantification of TUNEL-positive apoptotic cells, revealing significant post-irradiation increase (p < 0.001) via robust image processing and statistical analysis. Supervisor: Inna Kolesnikova


**On-site Undergraduate Research Student [Saveetha School of Engineering, India]**    
*Jan ’23 - Mar ’24*

* Under Kannan's supervision, conducted data preprocessing and exploratory data analysis (EDA) on the ChEMBL dataset for Alzheimer's drug discovery using pandas and matplotlib/seaborn
* Implemented and compared machine learning models (Random Forest, SVM, XGBoost) with scikit-learn, achieving 82% accuracy with Random Forest for predicting potential drug candidates.
* Performed molecular docking analysis using AutoDock Vina, applied Lipinski's rule of 5 with RDKit, and executed molecular dynamics simulations using GROMACS to validate and analyze protein-ligand interactions of promising Alzheimer's drug molecules. 

  
Skills
======
**Laboratory skills**   
Pipetting, Buffer, and Culture Media Preparation, Cell Culture, Plasmid and Genomic DNA Isolation, PCR, Gel Electrophoresis, Molecular Cloning, Thin Layer Chromatography, Assay of Biomolecules, Staining, Handling Equipment such as Nanodrop Spectrophotometer, X-Ray Crystallography, Cryogenic Electron Microscope, Colorimeter, Autoclave, and Centrifuge.

**Computational skills**   
Python (RDKit, Scikit, Numpy, Pandas, Matplotlib, Seaborn), R, Cytoscape, CellDesigner, PyRx, Chimera, PyMol, LigPlot along with MS Excel and Powerpoint 

Publications
======
  <ul>{% for post in site.publications reversed %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Talks
======
  <ul>{% for post in site.talks reversed %}
    {% include archive-single-talk-cv.html  %}
  {% endfor %}</ul>
  
Volunteering Experience
====
**Data Science Intern [Suvidha Foundation, Nagpur]**                                                                              
*Nov’23 - Dec’23*

* Implemented predictive modeling techniques to optimize resource allocation within the NGO, leading to a 25% reduction in project overhead costs and enhancing operational efficiency across all departments by streamlining resource distribution, and raised Rs. 2000 in additional funding for the organization.

**Teaching Intern [Jeevan Jagriti Foundation, Delhi]**                                                                                
*Sep’22 - Jan’23*

* Conducted online classes for underprivileged students using Google Meet, ensuring 80% attendance. Improved student pass rate to 70% and significantly boosted one student's final score from 70 to 90+.
* Led workshops for parents on student wellness and managed learning materials provided through donations, optimizing the educational experience for remote learners.


